tests/xharness/Targets/Target.cs

```diff
@@ -67,12 +70,55 @@ public abstract class Target
 
 		public string LanguageGuid { get { return IsFSharp ? FSharpGuid : CSharpGuid; } }
 
+		public abstract string DotNetSdk { get; }
+		public abstract string RuntimeIdentifier { get; }
+		public abstract DevicePlatform ApplePlatform { get; }
+		public abstract string TargetFrameworkForNuGet { get; }
+
+		public string PlatformString {
+			get {
+				switch (ApplePlatform) {
+				case DevicePlatform.iOS:
+					return "iOS";
+				case DevicePlatform.tvOS:
+					return "tvOS";
+				case DevicePlatform.watchOS:
+					return "watchOS";
+				case DevicePlatform.macOS:
+					return "macOS";
+				default:
+					throw new Exception ($"Unknown platform: {ApplePlatform}");
+				}
+			}
```

Comment:
What about moving this to an Extension method of ApplePlatform in case is useful in other cases?

Reply:
@mandel-macaque fixed